hw-model/src/lib.rs

```diff
@@ -629,7 +660,7 @@ pub trait HwModel {
                 "Fuses are already locked in place (according to cptra_fuse_wr_done)"
             );
         }
-        println!("Initializing fuses: {:#x?}", fuses);
+        //        println!("Initializing fuses: {:#x?}", fuses);
```

Comment:
I think we can leave this uncommented out because it is in the `std` file

Reply:
Done